bot/exts/evergreen/space.py

```diff
@@ -74,11 +74,18 @@ def cog_unload(self) -> None:
         # Parse date to params, when provided. Show error message when invalid formatting
         if date:
             try:
-                params["date"] = datetime.strptime(date, "%Y-%m-%d").date().isoformat()
+                apod_date = datetime.strptime(date, "%Y-%m-%d").date()
             except ValueError:
                 await ctx.send(f"Invalid date {date}. Please make sure your date is in format YYYY-MM-DD.")
                 return
 
+            now = datetime.now().date()
+            if datetime(1995, 6, 16).date() > apod_date or now < apod_date:
```

Comment:
Why not just build a date object directly?
```python
date(1995, 6, 16)
```
I'd probably make this specific date a constant and perhaps explain why it's needed, but it's not necessary.

Reply:
If this is made a constant (or at least a local variable), then it can be referenced in the error message for requests exceeding the allowed range.